contracts/ModuleRegistry.sol

```diff
@@ -296,7 +300,7 @@ contract ModuleRegistry is IModuleRegistry, EternalStorage {
     /**
      * @notice Called by the owner to pause, triggers stopped state
      */
-    function pause() external whenNotPaused onlyOwner {
+    function pause() external onlyOwner {
```

Comment:
I think `pause` should revert if the contract is already paused. It makes the logs easier to interpret. Can we have `whenNotPaused` and `whenNotPausedOrOwner` and use `whenNotPaused` here maybe?